src/main/java/com/conveyal/r5/analyst/cluster/GridResultAssembler.java

```diff
@@ -135,17 +132,17 @@ public void handleMessage (Message message) {
             Origin origin = Origin.read(bais);
 
             // if this is not the first request, make sure that we have the correct number of iterations
-            if (buffer != null && origin.accessibilityPerIteration.length != this.nIterations) {
+            if (buffer != null && origin.samples.length != this.nIterations) {
```

Comment:
I updated the comment to match "samples" but I think samples is a good term since these can be either instantaneous accessibility values or bootstrapped values, depending on the version of r5.